server/parsec/components/memory/user.py

```diff
@@ -692,7 +692,15 @@ async def test_dump_current_users(
                 devices=[
                     d.cooked.device_id for d in org.devices.values() if d.cooked.user_id == user_id
                 ],
-                current_profile=user.current_profile,
+                profile_updates=[
+                    # Initial profile
+                    (user.cooked.timestamp, user.cooked.profile),
+                    # Subsequent profile updates
+                    *(
+                        (update.cooked.timestamp, update.cooked.new_profile)
+                        for update in user.profile_updates
+                    ),
+                ],
```

Comment:
Not related to sequester, extract it to another PR? (see in `postgresql/user.py`)

Reply:
Moved in #9451